File: python/rich_menu_object.py
def rich_menu_object_a():
  return {
    "size": {
      "width": 2500,
      "height": 1686
    },
    "selected": False,
    "name": "richmenu-a",
    "chatBarText": "Tap to open",
    "areas": [
      {
        "bounds": {
          "x": 0,
          "y": 0,
          "width": 1250,
          "height": 1686
        },
        "action": {
          "type": "uri",
          "uri": "https://developers.line.biz/"
        }
      },
      {
        "bounds": {
          "x": 1251,
          "y": 0,
          "width": 1250,
          "height": 1686
        },
        "action": {
          "type": "richmenuswitch",
          "richMenuAliasId": "richmenu-alias-b",
          "data": "richmenu-changed-to-b"
        }
      }
    ]
  }

def rich_menu_object_b():
  return {
    "size": {
      "width": 2500,
      "height": 1686
    },
    "selected": False,
    "name": "richmenu-a",
    "chatBarText": "Tap to open",
    "areas": [
      {
        "bounds": {
          "x": 0,
          "y": 0,
          "width": 1250,
          "height": 1686
        },
        "action": {
          "type": "richmenuswitch",
          "richMenuAliasId": "richmenu-alias-a",
          "data": "richmenu-changed-to-a"
        }
      },
      {
        "bounds": {
          "x": 1251,
          "y": 0,
          "width": 1250,
          "height": 1686
        },
        "action": {
          "type": "uri",
          "uri": "https://developers.line.biz/"
        }
      }
    ]
  }

File: golang/rich_menu_object.go
package main

import "github.com/line/line-bot-sdk-go/v7/linebot"

func richMenuObjectA() linebot.RichMenu {
	return linebot.RichMenu{
		Size:        linebot.RichMenuSize{Width: 2500, Height: 1686},
		Selected:    false,
		Name:        "richmenu-a",
		ChatBarText: "Tap to open",
		Areas: []linebot.AreaDetail{
			{
				Bounds: linebot.RichMenuBounds{X: 0, Y: 0, Width: 1250, Height: 1686},
				Action: linebot.RichMenuAction{
					Type: linebot.RichMenuActionTypeURI,
					URI:  "https://linedevelopercommunity.connpass.com/event/223972/",
					Text: "click me",
				},
			},
			{
				Bounds: linebot.RichMenuBounds{X: 1251, Y: 0, Width: 1250, Height: 1686},
				Action: linebot.RichMenuAction{
					Type:            linebot.RichMenuActionTypeRichMenuSwitch,
					RichMenuAliasID: "richmenu-alias-b",
					Data:            "action=richmenu-changed-to-b",
				},
			},
		},
	}
}

func richMenuObjectB() linebot.RichMenu {
	return linebot.RichMenu{
		Size:        linebot.RichMenuSize{Width: 2500, Height: 1686},
		Selected:    false,
		Name:        "richmenu-a",
		ChatBarText: "Tap to open",
		Areas: []linebot.AreaDetail{
			{
				Bounds: linebot.RichMenuBounds{X: 0, Y: 0, Width: 1250, Height: 1686},
				Action: linebot.RichMenuAction{
					Type:            linebot.RichMenuActionTypeRichMenuSwitch,
					RichMenuAliasID: "richmenu-alias-a",
					Data:            "action=richmenu-changed-to-a",
				},
			},
			{
				Bounds: linebot.RichMenuBounds{X: 1251, Y: 0, Width: 1250, Height: 1686},
				Action: linebot.RichMenuAction{
					Type: linebot.RichMenuActionTypeURI,
					URI:  "https://linedevelopercommunity.connpass.com/event/223972/",
					Text: "click me",
				},
			},
		},
	}
}
